client/my-sites/themes/single-site-jetpack.jsx

```diff
@@ -143,13 +161,15 @@ export default connect(
 			const wpcomThemesCount = getThemesFoundForQuery( state, 'wpcom', wpcomQuery );
 			emptyContent = ( ! siteThemesCount && ! wpcomThemesCount ) ? null : <div />;
 		}
+		const currentPlan = getCurrentPlan( state, siteId );
 		return {
 			canManage: canJetpackSiteManage( state, siteId ),
 			hasJetpackThemes: hasJetpackSiteJetpackThemes( state, siteId ),
 			wpcomTier: hasFeature( state, siteId, FEATURE_UNLIMITED_PREMIUM_THEMES ) ? tier : 'free',
```

Comment:
You could drop this, since you're not using it anymore. Better yet however, what about dropping the `currenPlanSlug` prop (and [these comparisons](7d7fba9bc430b059b02739e9e281a237R98)), and replacing them with a 

```js
hasUnlimitedPremiumThemes: hasFeature( state, siteId, FEATURE_UNLIMITED_PREMIUM_THEMES )
```

prop? That would be more descriptive, too.